tests/test_vlan.rb

```diff
@@ -205,7 +205,7 @@ def test_vlan_add_remove_interface_valid
     interfaces_added_to_vlan = []
     count = 3
     interfaces.each do |name, interface|
-      next unless interface.name.match(%r{ethernet[0-9/]}) && count > 0
+      next unless interface.name.match(%r{ethernet[0-9/]+$}) && count > 0
```

Comment:
Actually I made this change specifically to exclude sub-interfaces - we can't set `switchport mode access` on sub-interfaces and I was encountering an error when running this test with an unclean base configuration that had sub-interfaces defined.